blobfuse/blobfuse.cpp

```diff
@@ -13,14 +13,14 @@ namespace {
 }
 
```

Comment:
Usage message is not updated with the new command line option: https://github.com/Azure/azure-storage-fuse/blob/ed79ab5108cbcb70dadc540e435d5622938c3d58/blobfuse/blobfuse.cpp#L201

Reply:
See next line - it was getting too long, so I split it up over two lines :)